packages/nc-gui/components/smartsheet/toolbar/OpenedViewAction.vue

```diff
@@ -0,0 +1,384 @@
+<script lang="ts" setup>
```

Comment:
some functions are exactly same in `ViewActions.vue`. May centralise them.

Reply:
This UX is still in experimental phase. After some time will analyse and see if ppl are also using the top bread crumb view info dropdown that much, if not might be removed. So we can pick up centralising this logic at that time